lib/Sema/BuilderTransform.cpp

```diff
@@ -1661,6 +1661,14 @@ Optional<BraceStmt *> TypeChecker::applyResultBuilderBodyTransform(
   // Build a constraint system in which we can check the body of the function.
   ConstraintSystem cs(func, options);
 
+  if (cs.isDebugMode()) {
+    auto &log = llvm::errs();
+
+    log << "--- Applying result builder to function ---\n";
```

Comment:
Nice, this will be super helpful in debug-constraints output 🙂

Reply:
Yeah, it's going to be a good indicator what is being processed piecemeal, otherwise it's sometimes unclear where the sub-targets are coming from...